context/app/static/js/components/detailPage/multi-assay/RelatedMultiAssayLinks/RelatedMultiAssayLinks.tsx

```diff
@@ -37,17 +39,22 @@ type Entries<T> = {
 interface MultiAssayLinkProps {
   dataset: MultiAssayEntity;
 }
-function MultiAssayLink({ dataset: { assay_display_name, uuid, hubmap_id } }: MultiAssayLinkProps) {
+function MultiAssayLink({ dataset: { assay_display_name, uuid, hubmap_id, status } }: MultiAssayLinkProps) {
   return (
-    <Typography>
-      {assay_display_name}: <InternalLink href={`/browse/dataset/${uuid}`}>{hubmap_id}</InternalLink>
-    </Typography>
+    <SecondaryBackgroundTooltip title={status}>
+      <Typography>
+        <FlexContainer>
+          {assay_display_name}:<InternalLink href={`/browse/dataset/${uuid}`}>{hubmap_id}</InternalLink>
+          <StatusIcon status={status} />
+        </FlexContainer>
```

Comment:
```suggestion
        <Stack direction="row" useFlexGap gap={0.5} alignItems="center">
          {assay_display_name}:<InternalLink href={`/browse/dataset/${uuid}`}>{hubmap_id}</InternalLink>
          <StatusIcon status={status} />
        </Stack>
```

The [MUI `Stack` component](https://mui.com/material-ui/react-stack/) is very useful in cases that call for a flex container on the go 😄